### PR TITLE
[LB-270] fix: removed ttl index on created_at from all collection

### PR DESCRIPTION
**Pull request by HtDeepMakadiya**
# Note
We will address the below comments later under this ticket
https://www.notion.so/hanabitech/Catalogue-Clean-up-collection-of-products-10405711b70f808ebe21e0cff46396a2?pvs=4

image

### CODE REVIEW

File: webserver/main/models/__init__.py [hunk 1]
```diff
@@ -37,42 +37,10 @@ def init_database():
...
 def create_all_indexes():
-    [create_ttl_index(c) for c in ["on_select", "on_init", "on_confirm", "on_cancel", "on_status", "on_support",
-                                   "on_track", "on_update", "on_rating"]]
-    [create_ttl_index(c, ttl_in_seconds=24*60*60) for c in ["on_search_dump", "request_dump", "on_search_items",
-                                                            "provider", "custom_menu", "location", "product",
-                                                            "product_attribute", "product_attribute_value",
-                                                            "variant_group", "customisation_group"
-                                                            ]]
     get_mongo_collection("on_search_items").create_index([('id', TEXT)], name='id_index')
     get_mongo_collection("location").create_index([("gps", GEOSPHERE)])
```
Comment (todaysolve) [inline webserver/main/models/__init__.py:41]: Only create if they are not present...

**Review by todaysolve — CHANGES_REQUESTED** [review]

File: webserver/main/config.py [hunk 2]
```diff
@@ -36,7 +36,6 @@ class Config:
     BAP_ID = os.getenv("BAP_ID", "buyer-app.ondc.org")
     BAP_UNIQUE_KEY_ID = os.getenv("BAP_UNIQUE_KEY_ID", "207")
     REGISTRY_BASE_URL = os.getenv("REGISTRY_BASE_URL", "https://staging.gateway.proteantech.in/ondc")
-    TTL_IN_SECONDS = int(os.getenv("TTL_IN_SECONDS", "18000"))
```
[the remark this answers is not in the document]
Comment (HtDeepMakadiya) [reply webserver/main/config.py:39]: Checked. Not present anywhere even in .env.example.

File: webserver/main/models/__init__.py (hunk 1 above)
Comment (HtDeepMakadiya) [reply webserver/main/models/__init__.py]: fixed

**Comment by todaysolve** [comment]
@HtDeepMakadiya please change ttll value to 7 days. But this PR approach is wrong, the database will always be outdated with network changes

**Review by todaysolve — APPROVED** [review]

**Pull request merged**